src/domain/app/footer/__tests__/Footer.test.tsx

```diff
@@ -1,10 +1,37 @@
 import * as React from 'react';
 
-import { render } from '../../../../util/testUtils';
+import { render, screen, userEvent } from '../../../../util/testUtils';
+import { ROUTES } from '../../routes/constants';
 import Footer from '../Footer';
 
 test('matches snapshot', () => {
   const { container } = render(<Footer />);
 
   expect(container.firstChild).toMatchSnapshot();
 });
+
+test.only('clicking links should route to right place', () => {
```

Comment:
Is the "only" there in a purpose?

Reply:
I use it only to get faster feedback when testing/debugging one test